README.md

```diff
@@ -37,7 +37,7 @@ If you select `WIFI_ESP8266`, `WIFI_ODIN` or `WIFI_RTW`, you also need to add th
 ```json
     "config": {
         "network-interface":{
-            "help": "options are ETHERNET, WIFI_ESP8266, WIFI_ODIN, WIFI_RTW, MESH_LOWPAN_ND, MESH_THREAD",
+            "help": "options are ETHERNET, WIFI_ESP8266, WIFI_ODIN, WIFI_RTW, MESH_LOWPAN_ND, MESH_THREAD,CELLULAR_ONBOARD",
```

Comment:
Please add a space after the `,`. This is inconsistent with the rest of the options.

Reply:
Spaces now made consistent.